Dockerfile

```
@@ -0,0 +1,7 @@
+# Use base image: https://github.com/keeganwitt/docker-gradle
+FROM gradle:jdk11
+
+RUN mkdir /app/atlas-checks
```

I had to alter this with the recursive create tag to get it to work. 
```suggestion
RUN mkdir -p /app/atlas-checks
```